l3kernel/l3doc.dtx

```diff
@@ -237,8 +237,10 @@ and all files in that bundle must be distributed together.
 % encoding (the standard setting) or leaves the font setup unchanged.
 %
 % \DescribeOption{kernel}
-% Determines whether \pkg{l3doc} treats functions and variables belong to
-% |kernel| module as allowable in code. In general,
+% Determines whether \pkg{l3doc} treats internal functions and variables belong
+% to |kernel| module as allowable in code, for instance
+% \cs{__kernel_tl_to_str:w}, \cs{c__kernel_expl_date_tl}, and
```

Comment:
my gut feeling is that it should say "or" not "and" in that case ... but what do I know :-)

Reply:
I think `and` is OK but `belong to |kernel|` should say `belonging to the |kernel|`

Reply:
Changed to `belonging to`, thanks @davidcarlisle.